tests/upgrade/upgrade_test.go

```diff
@@ -0,0 +1,134 @@
+//go:build upgrade
+
+package upgrade
+
+import (
+	"fmt"
+
+	cli "github.com/cheqd/cheqd-node/tests/upgrade/cli"
+	govtypesv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Upgrade - Execute", func() {
```

Comment:
This will need some changes but will mostly not required likely. Maybe just the voting bits are needed.